lib/import.js

```diff
@@ -25,8 +25,11 @@ const importIntoDB = (readFile, db) => {
 	const calendarDates = readFile('calendar_dates')
 	const pCalendarDates = convert(calendarDates, dataToDb.calendar_dates)
 
-	return settle([pCalendar, pCalendarDates])
-	.then(([calendarResult, calendarDatesResult]) => {
+	const shapes = readFile('shapes')
+	const pShapes = convert(shapes, dataToDb.shapes)
```

Comment:
Why not run this below, in parallel to `agency`/`stops`/etc.?

Reply:
If I understand the methods correctly `shapes` should be allowed to fail, like one of calendar/calendar_dates, the the files below shouldn't.

Reply:
True, didn't notice that. Parallelising more would still be beneficial, but I'll merge it anyways.